Kentor.AuthServices/SAML2P/Saml2Response.cs

```diff
@@ -473,6 +473,8 @@ private static void CheckSignature(XmlElement signedRootElement, IEnumerable<Asy
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate")]
         public IEnumerable<ClaimsIdentity> GetClaims(IOptions options)
         {
+            if (options == null)
+                throw new ArgumentNullException("options");
```

Comment:
https://github.com/KentorIT/authservices/blob/master/CONTRIBUTING.md requires `{}` even for single line `if`.